src/Avalonia.Controls.ColorPicker/Themes/Fluent/ColorPicker.xaml

```diff
@@ -43,39 +50,454 @@
           </DropDownButton.Content>
           <DropDownButton.Flyout>
             <Flyout FlyoutPresenterClasses="nopadding">
-              <ColorView x:Name="FlyoutColorView"
-                         Color="{Binding Color, RelativeSource={RelativeSource TemplatedParent}, Mode=TwoWay}"
-                         ColorModel="{Binding ColorModel, RelativeSource={RelativeSource TemplatedParent}, Mode=TwoWay}"
-                         ColorSpectrumComponents="{TemplateBinding ColorSpectrumComponents}"
-                         ColorSpectrumShape="{TemplateBinding ColorSpectrumShape}"
-                         HsvColor="{Binding HsvColor, RelativeSource={RelativeSource TemplatedParent}, Mode=TwoWay}"
-                         IsAccentColorsVisible="{TemplateBinding IsAccentColorsVisible}"
-                         IsAlphaEnabled="{TemplateBinding IsAlphaEnabled}"
-                         IsAlphaVisible="{TemplateBinding IsAlphaVisible}"
-                         IsColorComponentsVisible="{TemplateBinding IsColorComponentsVisible}"
-                         IsColorModelVisible="{TemplateBinding IsColorModelVisible}"
-                         IsColorPaletteVisible="{TemplateBinding IsColorPaletteVisible}"
-                         IsColorPreviewVisible="{TemplateBinding IsColorPreviewVisible}"
-                         IsColorSpectrumVisible="{TemplateBinding IsColorSpectrumVisible}"
-                         IsColorSpectrumSliderVisible="{TemplateBinding IsColorSpectrumSliderVisible}"
-                         IsComponentSliderVisible="{TemplateBinding IsComponentSliderVisible}"
-                         IsComponentTextInputVisible="{TemplateBinding IsComponentTextInputVisible}"
-                         IsHexInputVisible="{TemplateBinding IsHexInputVisible}"
-                         MaxHue="{TemplateBinding MaxHue}"
-                         MaxSaturation="{TemplateBinding MaxSaturation}"
-                         MaxValue="{TemplateBinding MaxValue}"
-                         MinHue="{TemplateBinding MinHue}"
-                         MinSaturation="{TemplateBinding MinSaturation}"
-                         MinValue="{TemplateBinding MinValue}"
-                         PaletteColors="{TemplateBinding PaletteColors}"
-                         PaletteColumnCount="{TemplateBinding PaletteColumnCount}"
-                         Palette="{TemplateBinding Palette}"
-                         SelectedIndex="{Binding SelectedIndex, RelativeSource={RelativeSource TemplatedParent}, Mode=TwoWay}">
-                <ColorView.Resources>
+
+              <!-- The following is copy-pasted from the ColorView's control template.
+                   It MUST always be kept in sync with the ColorView (which is master).
+                   Note the only changes are resources specific to the ColorPicker. -->
+              <Grid RowDefinitions="Auto,Auto">
```

Comment:
If there is any known way to avoid this duplication using `<Template>` or something in Avalonia I would like to know. However, I'm not currently aware of any other way to make this work.